microk8s-resources/actions/common/utils.sh

```diff
@@ -123,7 +123,7 @@ refresh_opt_in_local_config() {
     if $(grep -qE "^$opt=" $config_file); then
         run_with_sudo "$SNAP/bin/sed" -i "s@^$opt=.*@$replace_line@" $config_file
     else
-        run_with_sudo "$SNAP/bin/sed" -i "$ a $replace_line" "$config_file"
+        run_with_sudo "$SNAP/bin/sed" -i "1i$replace_line" "$config_file"
```

Comment:
Is this on purpose?

Reply:
Yes, this is on purpose. The way run_with_sudo works at the moment does not allow for spaces in the arguments passed.